WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1697,6 +1728,16 @@ private extension NotificationsViewController {
             }
         }
 
+        var confirmationMessageTitle: String {
+            switch self {
+            case .none:     return NSLocalizedString("all", comment: "Filter all confirmation tab name")
+            case .unread:   return NSLocalizedString("unread", comment: "Filter unread confirmation tab name")
+            case .comment:  return NSLocalizedString("comment", comment: "Filter comments confirmation tab name")
+            case .follow:   return NSLocalizedString("follow", comment: "Filter follows confirmation tab name")
+            case .like:     return NSLocalizedString("like", comment: "Filter likes confirmation tab name")
```

Comment:
Let's improve these `comment`s a bit. I don't think these would be clear to a translator what they're for. How about something like `Displayed in the confirmation alert when marking <type> notifications as read.`?